Spikes/Character.cpp

```diff
@@ -0,0 +1,143 @@
+#include "stdafx.h"
+#include "Character.h"
+
+
+
+//GLOBAL VARIABLE (FOR TESTING PURPOSES)
+const int PATH_RADIUS = 200;
+
+
+
+Character::Character(float radius, float speed, Color green, Color yellow, 
+					 Vector2D centre) 
+:	M_PI(std::acos(-1)),
+	DEGREES(M_PI / 180.0f),
```

Comment:
can declare M_PI and DEGREES like this at the top of the class
#define M_PI 3.141592654